src/Models/ShallowWaterModels/velocity_and_tracer_tendencies.jl

```diff
@@ -0,0 +1,50 @@
+using Oceananigans.Advection
+using Oceananigans.Coriolis
+using Oceananigans.Operators
+
+@inline function u_velocity_tendency(i, j, k, grid,
+                                     advection,
+                                     coriolis,
+                                     velocities,
+                                     tracers,
+                                     clock)
+
+# Put in h for hydrostatic pressure
+    return ( - ∂xᶠᵃᵃ(i, j, k, grid, hydrostatic_pressure) )
+end
+
+@inline function v_velocity_tendency(i, j, k, grid,
+                                     advection,
+                                     coriolis,
+                                     velocities,
+                                     tracers,
+                                     clock)
+
+# Put in h for hydrostatic pressure
+    return ( - ∂yᵃᶠᵃ(i, j, k, grid, hydrostatic_pressure) )
+
+end
+
+@inline function height_tendency(i, j, k, grid,
```

Comment:
💯

Reply:
In anticipation of the next step, I changed velocities to solution in lines 8 and 19.